package.json

```diff
@@ -72,7 +72,7 @@
     "update:test-devtools": "bash lighthouse-core/test/chromium-web-tests/test-locally.sh --reset-results",
     "test-devtools": "bash lighthouse-core/test/chromium-web-tests/test-locally.sh",
     "open-devtools": "bash lighthouse-core/scripts/open-devtools.sh",
-    "run-devtools": "bash lighthouse-core/test/chromium-web-tests/test-page-devtools.sh",
+    "run-devtools": "node lighthouse-core/scripts/pptr-run-devtools.js",
```

Comment:
shall we remove the old script or does it still serve a purpose?

Reply:
Yeah, I'm ok to remove.